src/expr.cpp

```diff
@@ -2086,10 +2095,11 @@ static ConstExpr *lConstFoldBinaryArithOp(BinaryExpr::Op op, const T *v0, const
     case BinaryExpr::Div:
```

Comment:
Div is handled here, while mod in lConstFoldBinaryIntOp(). This is inconsistent. This needs to be handle in the same place (either of them).

Reply:
That'd by design
lConstFoldBinaryIntOp : Constant fold the binary integer operations that aren't also applicable  to floating-point types. So, Mod goes here

lConstFoldBinaryArithOp - For those applicable for float. Hence div

Reply:
Ok, looks good.